packages/components-react/projects/uxpin-wrapper/src/form/SelectWrapperDummy/SelectWrapperDummy.tsx

```diff
@@ -4,7 +4,7 @@ import { partitionProps } from '../../form-utils';
 
 export type SelectProps = PSelectWrapperProps & DummySelectProps;
 
-export const Select = (props: SelectProps): JSX.Element => {
+export const SelectWrapperDummy = (props: SelectProps): JSX.Element => {
```

Comment:
Not sure if this will break existing prototypes? Perhaps an alias would be better or shouldn't this be called `SelectWrapper` since this in the end is a template?

Reply:
Theres already a `SelectWrapper` component generated as well. This can't be named Select since the `Select` will also be generated.